news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/salvation.jpg)](http://osu.ppy.sh/beatmapsets/840189)
+
+One of the most well-known mappers, [alienflybot](https://osu.ppy.sh/users/636114), once again brought excellence as well as challenges to fruit-enthusiaists, with his new marathon map on Yooh's "salvation". Combining aesthetics with intensive hyperdashes, anti-flows and occasional edge jumps, the map well expresses how this epic song goes from calm to hectic. If you are confident about your stamina and precision, this map would surely be the one you don't want to miss!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/dorchadas.jpg)](http://osu.ppy.sh/beatmapsets/812855)
+
+As an explorer of creative patterns in osu!catch, [Sinnoh](https://osu.ppy.sh/users/4236057) always led the way. A lot of extending sliders creating an entirely different gameplay experience were used in his new map on Rita's "dorchadas". Despite the sliders, don't miss the charming dash chains as well! With the fresh new patterns and smooth gameplay enhancement, Sinnoh created a fantastic fruit festival and proved that osu!catch mapping has more possibilities than we thought.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/heron.jpg)](http://osu.ppy.sh/beatmapsets/576786)
+
+S-C-U's "heron" is one of the most fun maps you can play if you enjoy catch! This set by [ZiRoX](https://osu.ppy.sh/users/200768), with a guest difficulty by [JBHyperion](https://osu.ppy.sh/users/4879508), has uncommon patterning and interesting mapping. The map fits perfectly with the song, is really enjoyable and challenging for anyone who plays it. If you want something new to check out, try this map and you'll surely enjoy it!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/wobble-impact.jpg)](http://osu.ppy.sh/beatmapsets/539769)
+
+An incredibly challenging set to play: Sota Fujimori's "Wobble Impact" is a song that never lets up with its pace. Constantly challenging the player with almost no breathing room. [Minato Yukina](https://osu.ppy.sh/users/531253)'s mapping will keep you on your toes as a wide variety of patterns are used to express the diverse set of wubs in the music.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/blonde-girl.jpg)](http://osu.ppy.sh/beatmapsets/715066)
+
+If you like energetic songs, then this is your perfect fit! JENNY ROM's "WWW.BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeat vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!
```

Comment:
I dont know if escaping the point in "WWW.BLONDE" will disable the automatic linking there, probably worth to try.

Reply:
This didn't help. Found another solution, might worth a try 👀